app.ts

```diff
@@ -41,6 +54,40 @@ export const getManifestFileName = (
   return "Clarinet.toml";
 };
 
+export const parseRemoteDataSettings = (
+  manifestPath: string,
+  radio: EventEmitter
+) => {
```

Comment:
What is the return type of this function? I think this should be renamed to `tryParseRemoteDataSettings` (more on `tryParseXyz` [here](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions-and-performance#try-parse-pattern)).

Reply:
We should add the return type.